must-gather/collection-scripts/common

```diff
@@ -9,7 +9,7 @@ get_env() {
   containers=$(oc -n $ns get po $pod -o jsonpath='{.spec.containers[*].name}')
   for container in $containers
   do
-    dockerfile=$(oc -n $ns exec $pod -c $container -- find /root/buildinfo -name $pattern || :)
+    dockerfile=$(oc -n $ns exec $pod -c $container -- find /root/buildinfo -name $pattern)
```

Comment:
Does this not break us on failure of this line?

Reply:
There's no set -e in these scripts. if the above line fails, dockerfile will be empty and the following if will skip over it.